### PR TITLE
feat: Add global message search

### LINKED ISSUE #243: As a business owner, I would like to search for conversations
- [ ] Search by any of the following attribute
  - Contact name 
  - Content in Messages
  - Agent Name

### PR DESCRIPTION
**Pull request by pranavrajs**
Fixes #243 

- Search by text for messages
te9bDtSRib

### CODE REVIEW

**Comment by pranavrajs** [comment]
@sojan-official Please disable template messages and case sensitivity in message search.

File: app/javascript/dashboard/routes/dashboard/conversation/search/Search.vue [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  text-align: left;
+  width: 100%;
+
+  &:hover {
+    background: var(--w-50);
+  }
+}
+
+.message-search--modal .modal-container {
+  width: 80rem;
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/conversation/search/Search.vue:160]: Move to `px`

**Review by nithindavid — APPROVED** [review]

**Review by nithindavid — APPROVED** [review]

**Comment by CL545740896** [comment]
i hope  it can locat  to my message in conversation , it is more friendly

**Comment by sojan-official** [comment]
@CL545740896  we will take it up in our upcoming releases.

**Pull request merged**